server-sq/server.js

```diff
@@ -55,17 +58,48 @@ app.post('/refresh', (req, res) => {
       res.sendStatus(400)
     })
 })
+*/
```

Comment:
I don't think we need this whole commented block anymore since we are not using a login or refreshing the token this way anymore. If we need it again later, we can go back into the previous commits instead of leaving this in our code base.